doc/tor.md

```diff
@@ -137,6 +137,12 @@ auth cookie will usually be `/run/tor/control.authcookie`:
 stat -c '%G' /run/tor/control.authcookie
 ```
 
+You can assign the Tor group to a variable called `TORGROUP`:
+
+```
+TORGROUP=$(stat -c '%G' /run/tor/control.authcookie)
+```
```

Comment:
Wouldn't it make more sense for this to replace the block above? Most people using a shell will be aware how to assign values to environment variables, but I do think it's a small improvement to add the assignment. But it doesn't need to be mentioned two times.

Reply:
Done.